Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -197,17 +197,17 @@ extension Module  {
 
     /// Converts build settings dictionary to a Plist object.
     ///
-    /// Adds string values in dictionaries as is and array values are quoted and then converted
-    /// to a string joined by whitespace.
+    /// Adds string values in dictionaries as is and array values are converted to a
+    /// string joined by whitespace.
     private func toPlist(_ buildSettings: [String: Any]) -> Plist {
         var buildSettingsPlist = [String: Plist]()
         for (k, v) in buildSettings {
             switch v {
             case let value as String:
                 buildSettingsPlist[k] = .string(value)
             case let value as [String]:
-                let escaped = value.map { "\"" + Plist.escape(string: $0) + "\"" }.joined(separator: " ")
-                buildSettingsPlist[k] = .string(escaped)
+                let joined = value.joined(separator: " ")
```

Comment:
we actually do need to quote because the build setting values may contain spaces (for eg spaces in path) which will be converted into arrays if not escaped. @abertelrud Do variables in format `$(identifier)` needs to be escaped with special rules? or the always quoting way is a disadvantage here?